rust/defs.bzl

```diff
@@ -130,6 +131,9 @@ rust_common = _rust_common
 rust_analyzer_aspect = _rust_analyzer_aspect
 # See @rules_rust//rust/private:rust_analyzer.bzl for a complete description.
 
+rust_analyzer_toolchain = _rust_analyzer_toolchain
```

Comment:
I'm just curious why export `rust_analyzer_toolchain` here instead of via `rust/toolchain.bzl` which exports `rust_toolchain`?

Reply:
I thought everything goes here. Do you think `rust_analyzer_toolchain` should be removed or `rust_toolchain` added?

Reply:
I suggest removing `rust_analyzer_toolchain` from https://github.com/bazelbuild/rules_rust/blob/main/rust/defs.bzl and adding it to https://github.com/bazelbuild/rules_rust/blob/main/rust/toolchain.bzl.

I can't find a `rust_toolchain` in https://github.com/bazelbuild/rules_rust/blob/main/rust/defs.bzl. Instead, it's defined in https://github.com/bazelbuild/rules_rust/blob/main/rust/toolchain.bzl. Since the `rust_analyzer_toolchain` is a toolchain, it feels more consistent to add it alongside of the the `rust_toolchain` definition.

Reply:
Moved this to `rust/toolchain.bzl`